plugins/plugin-git/che-plugin-git-ext-git/src/main/java/org/eclipse/che/ide/ext/git/client/compare/AlteredFiles.java

```diff
@@ -47,7 +48,7 @@ public AlteredFiles(Project project, String diff) {
         alteredFilesStatuses = new LinkedHashMap<>();
         if (!isNullOrEmpty(diff)) {
             for (String item : diff.split("\n")) {
-                if (item.length() < 3 || item.charAt(1) != ' ') {
+                if (item.length() < 3 || item.charAt(1) != '\t') {
                     throw new IllegalArgumentException("Invalid git diff format.");
```

Comment:
I think we should throw an exception here. It will be caught and a user will see it in the events section.
To me it will be better to say that something went wrong than show partial diff.